netbox/resource_netbox_device.go

```diff
@@ -107,6 +107,24 @@ func resourceNetboxDevice() *schema.Resource {
 				Type:     schema.TypeFloat,
 				Optional: true,
 			},
+			"virtual_chassis_id": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				RequiredWith: []string{"vc_master", "virtual_chassis_id"},
```

Comment:
I think you do not have to include the attribute itself when using `RequiredWith`.

Reply:
I thought it was required because the documentation [states](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema#Schema):

```
// RequiredWith is a set of attribute paths, including this attribute,
// that must be set simultaneously.
```

Reply:
Ok. Hm. It makes the docs a bit ugly, but youre right. Sorry for the inconvenience.